kong/conf_loader.lua

```diff
@@ -119,6 +119,8 @@ local CONF_INFERENCES = {
   pg_password = { typ = "string" },
   pg_ssl = { typ = "boolean" },
   pg_ssl_verify = { typ = "boolean" },
+  pg_semaphore_max = { typ = "number" },
+  pg_semaphore_timeout = { typ = "number" },
```

Comment:
I suggest ensuring that none of these values can be negative, which would result in failures to initialize the semaphore. Along with updates to the `conf_loader` unit tests suite to ensure that is the case.

Reply:
Ah yeah, I missed the `conf_loader` changes. Fixed up!